composer.json

```diff
@@ -20,7 +20,7 @@
         "drupal/drupal-extension": "^4.0.0@alpha",
         "drush/drush": "^9",
         "nikic/php-parser": "~3",
-        "openeuropa/code-review": "^0.2",
+        "openeuropa/code-review": "^0.3",
```

Comment:
Let's use `~0.3` as the version range so that when a new version of the code review tool is released, we can automatically use it without having to update the composer file again.